test/dns_nsupdate/dns_nsupdate_test.rb

```diff
@@ -1,8 +1,6 @@
 require 'test_helper'
-require 'dns/dns'
-require 'dns_nsupdate/dns_nsupdate_plugin'
+require 'dns_common/dns_common'
 require 'dns_nsupdate/dns_nsupdate_main'
-require 'dns_nsupdate/dns_nsupdate_gss_main'
```

Comment:
I'll add these lines(all 3?), but I'd to change this header, cause my local tests did not run at all!